OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -168,7 +169,7 @@ public virtual void SelectTarget(Actor self, string order, SupportPowerManager m
 
 		public virtual void Activate(Actor self, Order order, SupportPowerManager manager)
 		{
-			if (Info.DisplayRadarPing && manager.RadarPings != null)
+			if (Info.DisplayRadarPing && manager.RadarPings.IsValueCreated)
```

Comment:
Why are you using `IsValueCreated` here and in other parts of the PR?

Reply:
See https://github.com/OpenRA/OpenRA/pull/17719#discussion_r489369099

Reply:
`Lazy<T>.IsValueCreated` is not equivalent to `Nullable<T>.HasValue`!

This change will stop the radar pings from ever being shown, because this is the only code path that can create the value.

This original intention for this check would have been `manager.RadarPings.Value != null` (which is the equivalent of `Nullable<T>.HasValue`), but like I said in the other comment, this is all bogus anyway because this shouldn't be using `Lazy` in the first place!